data/products.json

```diff
@@ -273,5 +273,51 @@
         "link": "https://supabase.com/"
       }
     ]
+  },
+  {
+    "name": "Deel",
+    "country_code": "US",
+    "alternatives": [
+      {
+        "name": "RemotePass",
+        "country_code": "AE",
+        "link": "https://www.remotepass.com/"
+      }
+    ]
+  },
+  {
+    "name": "Wiz",
+    "country_code": "US",
+    "alternatives": [
```

Comment:
Thanks for the suggestion! However, we're currently focusing on other priorities and won't be pursuing US-based alternatives at this time.